src/components/document/document-workspace.tsx

```diff
@@ -52,19 +65,22 @@ export class DocumentWorkspaceComponent extends BaseComponent<IProps> {
           onDragOver={this.handleDragOverWorkspace}
           onDrop={this.handleImageDrop}
         />
-        {this.renderDocuments()}
-        <NavTabPanel
-          tabs={tabsToDisplay}
-          isTeacher={isTeacher}
-          onDragOver={this.handleDragOverWorkspace}
-          onDrop={this.handleImageDrop}
-        />
-        <NavTabButtons
-          tabs={tabsToDisplay}
-          isTeacher={isTeacher}
-          onDragOver={this.handleDragOverWorkspace}
-          onDrop={this.handleImageDrop}
-        />
+
+        {navTabContentShown
+          ? <NavTabPanel
+              tabs={tabsToDisplay}
+              isTeacher={isTeacher}
+              onDragOver={this.handleDragOverWorkspace}
+              onDrop={this.handleImageDrop}
+            />
+          : <CollapsedResourcesTab onExpandResources={this.setExpandResources} resourceType={activeNavTab} />
+        }
+        <ResizePanelDivider isResourceExpanded={navTabContentShown} onExpandWorkspace={this.setExpandWorkspace}/>
+        {this.state.expandWorkspace ? this.renderDocuments()
+                                    : <CollapsedWorkspaceTab onExpandWorkspace={this.setExpandWorkspace}
+                                                             workspaceType={type}
+                                      />
```

Comment:
I would probably write this as the close tag bracket confused me, but it doesn't really matter:
```
        {this.state.expandWorkspace ? this.renderDocuments()
                                    : <CollapsedWorkspaceTab 
                                        onExpandWorkspace={this.setExpandWorkspace}
                                        workspaceType={type}
                                      />
```